packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -145,18 +150,22 @@ class _CodeViewState extends State<CodeView>
       log('LinkedScrollControllerGroup has no attached controllers');
       return;
     }
-
     final location = widget.controller.scriptLocation.value?.location;
     if (location?.line == null) {
-      // Default to scrolling to the top of the script.
-      if (animate) {
-        verticalController.animateTo(
-          0,
-          duration: longDuration,
-          curve: defaultCurve,
-        );
-      } else {
-        verticalController.jumpTo(0);
+      // Don't scroll to top if we're just rebuilding the code view for the
+      // same script.
+      if (_lastScriptRef?.uri != scriptRef?.uri) {
```

Comment:
will this still scroll properly when selecting different members for the same file from the outline view?

Reply:
Yup. This just makes sure that we don't try and reset the scroll position if we're just rebuilding the code view for the same script. When we select a member from the outline view, we don't actually trigger a rebuild of the code view (AFAIK).